data_steward/utils/participant_summary_requests.py

```diff
@@ -157,6 +172,73 @@ def get_deactivated_participants(project_id, dataset_id, tablename, columns):
     return df
 
 
+def get_site_participant_information(project_id, hpo_id):
+    """
+    Fetches the necessary participant information for a particular site.
+
+    :param project_id: The RDR project that contains participant summary data
+    :param hpo_id: awardee name of the site
+
+    :return: returns dataframe of participant information
+    """
+    # Parameter checks
+    if not isinstance(project_id, str):
+        raise RuntimeError(f'Please specify the RDR project')
+
+    if not isinstance(hpo_id, str):
+        raise RuntimeError(f'Please provide an hpo_id')
+
+    token = get_access_token()
+
+    headers = {
+        'content-type': 'application/json',
+        'Authorization': 'Bearer {0}'.format(token)
+    }
+
+    # Make request to get API version. This is the current RDR version for reference see
+    # see https://github.com/all-of-us/raw-data-repository/blob/master/opsdataAPI.md for documentation of this API.
+    url = 'https://{0}.appspot.com/rdr/v1/ParticipantSummary?awardee={1}&_sort=participantId&_count=1000'.format(
+        project_id, hpo_id)
```

Comment:
you can break this into a two lines and using f-strings for readability like
```suggestion
    url = (f'https://{project_id}.appspot.com/rdr/v1/ParticipantSummary'
              # get all participants for a site 1000 at a time.  improves performance
              f'?awardee={hpo_id}&_sort=participantId&_count=1000')
```
This will let you insert comments into the string explaining why fields and values were chosen.

Also, does this successfully get all data for a site with more than 1000 participants?

Reply:
Also, this looks like it is retrieving all data for each participant it is getting from the API.  Is there a way to use the API to only retrieve the fields we are interested in?  Not sure from the documentation yet.  This would reduce the performance time on the system too.

Reply:
I will look into it. Only the columns we are interested in are inserted into the dataframe in the end but if I will see if we can limit the columns on the front end!

Reply:
I will also double check that it's getting all the data and not limiting to 1000. At first I thought it would receive a chunk of 1000 participants per URL and then return them all in the end but now I'm not too sure so I will double check!

Reply:
Added a counter and printed out the participants and made sure that all were being returned. 
<img width="1389" alt="Screen Shot 2020-12-15 at 9 42 05 AM" src="https://user-images.githubusercontent.com/62044448/102237369-380ac080-3eba-11eb-8b36-61efb27ea3fa.png">

Still looking to see if we can limit to just retrieve info from the columns that we want!

Reply:
Kenny Skaggs indicated the participant summary api does not currently let us specify the fields we are interested in.  It seems to be an all or nothing thing.

What does the above indicate?  Did it return 32751 records?